doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -4962,19 +4960,19 @@ bookkeeping steps.
 
   thus in this case, the tactic ``apply/PQequiv`` is equivalent to
   ``apply: (iffRL (PQequiv _ _))``, where ``iffRL`` is the analogue of
-  ``iffRL`` for the converse implication.
+  ``iffLR`` for the converse implication.
 
 Any |SSR| term whose type coerces to a double implication can be
 used as a view for goal interpretation.
 
-Note that the goal interpretation view mechanism supports both ``apply``
-and ``exact`` tactics. As expected, a goal interpretation view command
-exact/term should solve the current goal or it will fail.
+Note that the goal-interpretation view mechanism supports both ``apply``
```

Comment:
no dash

Reply:
See above

Reply:
Please fix.  Not common usage.  The doc uses "goal interpretation" as a noun; this is a noun phrase.